sentry-tracing/src/converters.rs

```diff
@@ -145,5 +173,37 @@ where
     // proper grouping and issue metadata generation. tracing_core::Record does not contain sufficient
     // information for this. However, it may contain a serialized error which we can parse to emit
     // an exception record.
-    event_from_event(event, ctx)
+    const DEFAULT_ERROR_VALUE_KEY: &str = "error";
+
+    let (message, mut visitor) = extract_event_data(event);
+
+    let exception = if visitor.exceptions.is_empty() {
+        let ty = event.metadata().name().into();
+        let value = visitor
+            .json_values
+            .remove(DEFAULT_ERROR_VALUE_KEY)
+            .map(|v| v.as_str().map(|s| s.to_owned()))
+            .flatten()
+            .or_else(|| message.clone());
```

Comment:
Is this whole thing still needed now that you discovered `record_error`?

Reply:
I left this branch as a fallback in the case that the event does not have any exceptions. This would at least create a single exception using what data is available from either the `error` or `message` field (in that order).